File: eDentist/UI/edentist_mobile/lib/models/medical_card.dart
import 'package:json_annotation/json_annotation.dart';

part 'medical_card.g.dart';

@JsonSerializable()
class MedicalCard {
  int medicalCardId;
  int userId;
  int? dentalServiceId;
  String? doctorsOppinion;
  DateTime datumDijagnoze;

  MedicalCard(this.medicalCardId, this.userId, this.dentalServiceId,
      this.doctorsOppinion, this.datumDijagnoze);

  factory MedicalCard.fromJson(Map<String, dynamic> json) =>
      _$MedicalCardFromJson(json);

  Map<String, dynamic> toJson() => _$MedicalCardToJson(this);
}

File: eDentist_rabbitmq/eDentist_rabbitmq/Database/MedicalCard.cs
using System;
using System.Collections.Generic;

namespace eDentist.Services.Database;

public partial class MedicalCard
{
    public int MedicalCardId { get; set; }

    public int? UserId { get; set; }

    public int? DentalServiceId { get; set; }

    public string? DoctorsOppinion { get; set; }
    public DateTime datumDijagnoze { get; set; }
    public virtual DentalService? DentalService { get; set; }

    public virtual User? User { get; set; }
}
